backend/protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -458,6 +458,9 @@ enum SecretProvider {
 
   // Store a secret in the AWS Secrets Manager.
   SECRET_ASM = 4;
+
+  // Use the database as a secret store.
+  SECRET_DB = 5;
```

Comment:
I think there's been a misunderstanding. We only want the DB to be used as a Router, not a Provider. The Router just stores a URI that is then used to route to the actual provider, and the provider retrieves it.

Reply:
Ahhh that makes sense! I thought the db was replacing the ASM store for some use cases like ftl box. On it.